specification/schemas/Shipment.json

```diff
@@ -115,23 +115,7 @@
             "options": {
               "type": "array",
               "items": {
-                "type": "object",
-                "properties": {
-                  "code": {
-                    "type": "string",
-                    "example": "delivery-window:sunday"
-                  },
-                  "name": {
-                    "type": "string",
-                    "example": "Sunday Delivery"
-                  },
-                  "values": {
-                    "type": "object",
-                    "example": {
-                      "pin": "1234"
-                    }
-                  }
-                }
+                "$ref": "#/components/schemas/ServiceOption"
```

Comment:
The ServiceOption schema has `code` listed as `required`. The POST /shipments spec on develop also already has it defined as `required`, even though the removed code from this Shipment schema doesn't list it as required. That probably means that there's something in the `paths/Shipments.json` that lists the `code` of options as required. Can you remove it there? 

![image](https://github.com/MyParcelCOM/carrier-specification/assets/32866117/efaeae94-c418-4416-baa0-e42ad049922d)

Reply:
done